static/js/utils/data_commons_client.ts

```diff
@@ -31,3 +31,12 @@ export const defaultDataCommonsWebClient = new DataCommonsWebClient({
 export const defaultDataCommonsClient = new DataCommonsClient({
   apiRoot: DEFAULT_CLIENT_API_ROOT,
 });
+
```

Comment:
supernit: Could you add a quick docstring for this exported function? Would be helpful to clarify why we need this function and can't just use `new DataCommonsClient({apiRoot})` directly.

Reply:
done!